### PR TITLE
Keep track of the total number of tracks in an album.

### PR DESCRIPTION
**Pull request by brentwalther**
Fixes Issue #77.

Sorry about the long diff. My editor removed a bunch of trailing whitespace. Will redo commit with a minimal diff if you insist.

### CODE REVIEW

**Comment by brentwalther** [comment]
Friendly ping.

File: lib/rspotify/album.rb [hunk 1]
```diff
@@ -84,10 +84,14 @@ def initialize(options = {})
         options['artists'].map { |a| Artist.new a }
       end
 
-      @tracks_cache = if options['tracks'] && options['tracks']['items']
-        options['tracks']['items'].map { |i| Track.new i }
+      @tracks_cache, @total = if options['tracks'] && options['tracks']['items']
```
Comment (guilhermesad) [inline lib/rspotify/album.rb]: Can you change `@total` for `@total_tracks`?

File: lib/rspotify/album.rb [hunk 2]
```diff
@@ -84,10 +84,14 @@ def initialize(options = {})
...
+        [
+          options['tracks']['items'].map { |i| Track.new i },
+          options['tracks']['total']
+        ]
       end
 
+
```
Comment (guilhermesad) [inline lib/rspotify/album.rb]: Extra line.

**Comment by guilhermesad** [comment]
Looks good, thanks! Can you add it to the docs in [here](https://github.com/guilhermesad/rspotify/blob/master/lib/rspotify/album.rb#L3-L13)? After that and the comments above I'll merge it.

**Comment by brentwalther** [comment]
Addressed comments.

**Comment by guilhermesad** [comment]
:+1:

**Pull request merged**